modules/sdk-core/src/bitgo/lightning/iLightning.ts

```diff
@@ -128,11 +133,28 @@ export const GetBalanceResponse = t.strict(
 // eslint-disable-next-line no-redeclare
 export type GetBalanceResponse = t.TypeOf<typeof GetBalanceResponse>;
 
+const InvoiceInfo = t.strict({
+  paymentHash: t.string,
+  walletId: t.string,
+  status: t.string,
+  value: t.number,
+  expiresAt: t.string,
+  createdAt: t.string,
+  updatedAt: t.string,
+  amtPaidSats: t.number,
+});
+
+export const GetInvoicesResponse = t.array(InvoiceInfo);
+
+// eslint-disable-next-line no-redeclare
```

Comment:
let's disable this in the `eslintrc` config

Reply:
`eslintrc` for `sdk-core`?

Reply:
@mmcshinsky-bitgo mentioned in my earlier pr that he doesn't prefer to disable`no-redeclare` for the file since this pattern is only specific to `io-ts`

Reply:
fair enough